src/tactic/ui/tools/ingest_wdg.py

```diff
@@ -1566,6 +1593,16 @@ def execute(my):
                 mode = "multi"
                 new_filename = filename
 
+            if library_mode:
+                
+                # get count of number of files in the current asset ingest dir
+                import glob
+                abs_path = Environment.get_asset_dir() + "/" + relative_dir + "/*"
+
+                if len(glob.glob(abs_path)) > FOLDER_LIMIT:
+                    current_folder = current_folder + 1
+                    relative_dir = relative_dir[:-4] + "/%03d" % current_folder
+
```

Comment:
Doesn't matter. The folder files get ingested into will just have less than 500 files.